apps/meteor/server/methods/sendForgotPasswordEmail.ts

```diff
@@ -20,7 +20,7 @@ Meteor.methods<ServerMethods>({
 
 		const email = to.trim().toLowerCase();
 
-		const user = await Users.findOneByEmailAddress(email, { projection: { _id: 1 } });
+		const user = await Users.findOneByEmailAddress(email, { projection: { _id: 1, services: 1 } });
```

Comment:
@MarcosSpessatto wdyt of changing findOneByEmail to accept a type parameter, so this projection is properly typed? 😬 

Know its a bit out of the scope, but would be nice to have

Reply:
Or maybe add it as a separate task, that works as well 😬

Reply:
yup, for a separate task, looks good

Reply:
@matheusbsilva137 , can u create one for that? 😬

Reply:
Sure!